lib/cli/watch.js

```diff
@@ -17,9 +18,14 @@ export default async function watch(options = {}) {
 
   const server = new Server(reptar);
   await server.start();
-
   activity.end(startActivity);
 
   process.stdout.write('\n');
   log.info('Server running at:', server.server.info.uri);
+
+  const browserSync = bsCreate();
+  browserSync.init({
+    files: `${reptar.config._raw.path.source}/**/*`,
```

Comment:
No problem! I was also thinking about a CLI flag. I pretty much always want to use browser-sync  but I could imagine a scenario where someone might only want to watch. What should the flag be? `-bs`, `-l`?